### ISSUE TITLE
Flameshot still doesn't work under gnome 41.3

### ISSUE THREAD

**Issue by lxwulf** [flameshot-org/flameshot#2351]
### Flameshot Version

Flameshot v11.0.0 (-)
Compiled with Qt 5.15.2
linux: 5.15.18-200.fc35.x86_64
fedora: 35

### Installation Type

Linux, MacOS, or Windows Package manager (apt, pacman, eopkg, choco, brew, ...)

### Operating System type and version

Fedora 35

### Description

If I execute `flameshot gui` it takes just a screenshot from my full desktop and then a notification pop-up appears which says `flameshot error: unable to capture screen`....

### Steps to reproduce

1. install `flameshot gui` over package manager, in my case `dnf`
2. execute flameshot gui in terminal or click on the icon and "Take screenshot" or "Open launcher"
3. After screens "flashes" the "screenshot menu" from gnome appears and at the same time the error notification appears from flameshot

### Screenshots or screen recordings

![Screenshot from 2022-02-02 07-45-22](https://user-images.githubusercontent.com/44492381/152106730-2f3ff026-bd8f-43e5-9f19-68a72aea8bd1.png)

### System Information

![Screenshot from 2022-02-02 07-47-21](https://user-images.githubusercontent.com/44492381/152106855-45cb5be8-d02b-4c46-adb8-36ff7391afb3.png)

![Screenshot from 2022-02-02 07-48-06](https://user-images.githubusercontent.com/44492381/152106913-d8a7c69c-59d3-40df-91f1-4e285c0ddfed.png)

Using Wayland

**Comment by mmahmoudian** [comment]
Do you by any chance have unicode characters in your `XDG_PICTURES_DIR`? I'm asking because of https://github.com/flameshot-org/flameshot/issues/2211

**Comment by lxwulf** [comment]
Yes, i thought you would ask but no i didn't have it. Sorry, i missed to mentioned it i my issue.

**Comment by mmahmoudian** [comment]
I'm not 100% sure if you would need them but just to be clear, can you check if you have the `xdg-desktop-portal` and `xdg-desktop-portal-gnome` installed?

**Comment by lxwulf** [comment]
Yes, i have it. Sorry for the late answer.

![Screenshot from 2022-02-03 14-32-19](https://user-images.githubusercontent.com/44492381/152352601-8595d4d3-52cb-4918-9ff8-41c8d4ee694a.png)

**Comment by borgmanJeremy** [comment]
The best way you can help resolve this issue is to find a way to reproduce it in a clean VM (or clean install). The problem is it works for all the dev's so there really isnt anything we can do to assist.

**Comment by lxwulf** [comment]
I agree. I tried it out – it worked fine. But I also did a reinstallation of flameshot in my machine, which didn't help. Are there some configuration files left over which I could delete?

**Comment by mmahmoudian** [comment]
@lxwulf
> Are there some configuration files left over which I could delete?

These are what I got

```console
fd --exclude "tmp/" --exclude ".local/" --exclude "Pictures/" --exclude ".mozilla/" --hidden flameshot
.cache/flameshot
.cache/tldr/pages/linux/flameshot.md
.cache/paru/clone/flameshot-git
.config/flameshotrc
.config/flameshot
.config/flameshot/flameshot.ini
```

You can safely ignore whatever in my `.cache/`

**Comment by mmahmoudian** [comment]
We don't have a good logging or verbosity implemented unfortunately. However, you can try killing all instances of Flameshot, then open two terminals, in one run `flameshot` and in the other `flameshot gui`. It output certain errors in the first terminal.

Anyways, I would also like to suggest you to try the AppImage as well and see if that works. It can points to the direction of missing some sort of dependency.

**Comment by lxwulf** [comment]
@borgmanJeremy @mmahmoudian 

Update: Also, the flatpak version doesn't work anymore at my system, at least. Would be there any difference when I build the application from source?

Greets LxWulf

**Comment by mmahmoudian** [comment]
> Would be there any difference when I build the application from source?

I don't think building from source would help, but trying it out would only take about 2 minutes, so nothing to lose there. You can find instructions here:

https://flameshot.org/docs/installation/source-code/

Also try the AppImage.

This issue is peculiar. We don't know under what circumstances it didn't work on your computer and we cannot replicate it.

**Comment by lxwulf** [comment]
Somehow that didn't work either. By chance, I had to reinstall my Fedora system anyway, and again I don't want to have to do without flameshot. So on which should I install it? What is the best method? Flatpak? DNF? Appimage?…

Greets LxWulf


PS: Yes, after that, you will never be heard from me on that specific ticket.😉

**Comment by mmahmoudian** [comment]
I art least can't tell. You can try Flatpak. The thing is Wayland support is not fully there yet.
